src/flare_pp/bffs/sparse_gp.cpp

```diff
@@ -1039,10 +1061,12 @@ SparseGP ::compute_likelihood_gradient(const Eigen::VectorXd &hyperparameters) {
   double sigma_f = hyperparameters(hyp_index + 1);
   double sigma_s = hyperparameters(hyp_index + 2);
 
-  Eigen::VectorXd noise_vec = sigma_e * sigma_e * e_noise_one + sigma_f * sigma_f * f_noise_one + sigma_s * sigma_s * s_noise_one; 
-  Eigen::VectorXd e_noise_grad = 2 * sigma_e * e_noise_one;
-  Eigen::VectorXd f_noise_grad = 2 * sigma_f * f_noise_one;
-  Eigen::VectorXd s_noise_grad = 2 * sigma_s * s_noise_one;
+  Eigen::VectorXd noise_vec = sigma_e * sigma_e * inv_e_noise_one 
+                            + sigma_f * sigma_f * inv_f_noise_one 
```

Comment:
noise_one variables changed to inv_noise_one. im sure you have confirmed that this is but the inconsistency (with previous naming) is why I am commenting

Reply:
yes, so basically I have kept both `e_noise_one` and `inv_e_noise_one`. Previously without relative noise, this value (the relative noise) is just 1. And now I use those two to save `rel_e_noise` and `1/rel_e_noise`.